### PR TITLE
Add an option [--skip-check-length-of-identifier] to skip check the length of identifier when generate entity

### PR DESCRIPTION
**Pull request by develguys**
Add an option [--skip-check-length-of-identifier] to skip check the length of identifier when generate entity

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by deepu105** [comment]
@develguys whats happening with all those PRS

**Comment by develguys** [comment]
@deepu105 Those PRS were deleted because it did not pass the test with travis

**Comment by deepu105** [comment]
You don't have to delete PR for that, you can commit to the same branch and
push updates to fix issues

**Comment by develguys** [comment]
@deepu105 Thanks, I'll keep that in mind

**Comment by jdubois** [comment]
Tested & merged! Congrats for your first PR @develguys !!

File: generators/generator-base.js [hunk 1]
```diff
@@ -2052,11 +2053,11 @@ module.exports = class extends PrivateBase {
         }
         let limit = 0;
 
-        if (prodDatabaseType === 'oracle' && constraintName.length > 30) {
+        if (prodDatabaseType === 'oracle' && constraintName.length > 30 && !this.skipCheckLengthOfIdentifier) {
             this.warning(`The generated constraint name "${constraintName}" is too long for Oracle (which has a 30 characters limit). It will be truncated!`);
 
             limit = 28;
-        } else if (prodDatabaseType === 'mysql' && constraintName.length > 64) {
+        } else if (prodDatabaseType === 'mysql' && constraintName.length > 64 && !this.skipCheckLengthOfIdentifier) {
```
Comment (pascalgrimaud) [inline generators/generator-base.js]: @jdubois @deepu105 : I think I found the root cause of https://github.com/jhipster/generator-jhipster/pull/7489

Comment (jdubois) [reply generators/generator-base.js]: oh yes that's the change I was referring to, it wasn't meant to have side effects

**Pull request merged**